rotkehlchen/tests/unit/decoders/test_convex.py

```diff
@@ -0,0 +1,1120 @@
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import (
+    HistoryBaseEntry,
+    HistoryEventSubType,
+    HistoryEventType,
+)
+from rotkehlchen.assets.asset import EthereumToken
+from rotkehlchen.chain.ethereum.decoding import EVMTransactionDecoder
+from rotkehlchen.chain.ethereum.decoding.constants import CPT_GAS
+from rotkehlchen.chain.ethereum.modules import Convex
+from rotkehlchen.chain.ethereum.modules.convex.constants import CPT_CONVEX
+from rotkehlchen.chain.ethereum.modules.convex.decoder import BOOSTER
+from rotkehlchen.chain.ethereum.structures import EthereumTxReceipt, EthereumTxReceiptLog
+from rotkehlchen.chain.ethereum.types import string_to_ethereum_address
+from rotkehlchen.constants import ONE, ZERO
+from rotkehlchen.constants.assets import A_CRV, A_CVX, A_ETH
+from rotkehlchen.db.ethtx import DBEthTx
+from rotkehlchen.fval import FVal
+from rotkehlchen.inquirer import Inquirer
+from rotkehlchen.types import EthereumTransaction, Location, deserialize_evm_tx_hash
+from rotkehlchen.user_messages import MessagesAggregator
+from rotkehlchen.utils.hexbytes import hexstring_to_bytes
+
+
+def test_convex_balances(database, ethereum_manager):
+    msg_aggregator = MessagesAggregator()
+    Inquirer().inject_ethereum(ethereum_manager)
+    convex = Convex(
+        database=database,
+        ethereum_manager=ethereum_manager,
+        premium=None,
+        msg_aggregator=msg_aggregator,
+    )
+    # addr1 and addr2 test staking of 2 addresses in the same pool (cvxCRV/CRV in this case)
+    addr1 = string_to_ethereum_address('0x0633F1692Dab82F6F253eF571C769861CF90ecb6')
+    addr2 = string_to_ethereum_address('0xC49Eb99c132795F74b3d6f71B2374dC35015d473')
+    # addr3 tests staking in multiple pools
+    addr3 = string_to_ethereum_address('0xd6ffc26607A60A0c5383533fd53F7C47c26D62D4')
+    user_addresses = [addr1, addr2, addr3]
+    found_balances = convex.get_balances(addresses=user_addresses)
+
+    assert len(found_balances[addr1]) == 2
+    assert len(found_balances[addr2]) == 2
+    assert len(found_balances[addr3]) == 2
```

Comment:
This isn't a test for the decoders. Move it to a different place as we do for other modules

Reply:
Done :+1: